src/ec/packages/ec-component-accordion2/accordion2.story.js

```diff
@@ -1,36 +1,51 @@
 /* eslint-disable no-param-reassign */
 import { storiesOf } from '@storybook/html';
-import { withKnobs, text } from '@storybook/addon-knobs';
+import { withKnobs, text, select } from '@storybook/addon-knobs';
 import { withNotes } from '@ecl-twig/storybook-addon-notes';
+import { buttonLabels, getExtraKnobs } from '@ecl-twig/story-utils';
 import withCode from '@ecl-twig/storybook-addon-code';
-
 import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
 import demoData from './demo/data';
 
 import accordion2 from './ecl-accordion2.html.twig';
 import notes from './README.md';
 
+const prepareAccordion2 = data => {
+  data.items.forEach((item, index) => {
+    const { id, level, toggle, content } = item;
+
+    item.id = select(`item[${index}].id`, [id], id, buttonLabels.required);
+    item.level = select(
+      `item[${index}].level`,
+      [level],
+      level,
+      buttonLabels.required
+    );
+    item.toggle.label = text(
```

Comment:
So, nice that yo uadded the missing knobs, but i think we can improve the order, logically i would expect the "items[i].content" to come earlier than the "items[i].toggle.icon.path", no?

Reply:
Sure, but i the ordered them the same the order from the notes